src/pages/home/report/ReportActionItemThread.js

```diff
@@ -24,43 +26,56 @@ const propTypes = {
     /** ID of child thread report */
     childReportID: PropTypes.string.isRequired,
 
-    /** localization props */
+    /** Whether the thread item / message is being hovered */
+    isHovered: PropTypes.bool.isRequired,
+
     ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
 };
 
-const ReportActionItemThread = (props) => (
-    <View style={[styles.chatItemMessage]}>
-        <Pressable
-            onPress={() => {
-                Report.openReport(props.childReportID);
-                Navigation.navigate(ROUTES.getReportRoute(props.childReportID));
-            }}
-        >
-            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2]}>
-                <MultipleAvatars
-                    size={CONST.AVATAR_SIZE.SMALLER}
-                    icons={props.icons}
-                    shouldStackHorizontally
-                    avatarTooltips={_.map(props.icons, (icon) => icon.name)}
-                />
-                <View style={[styles.flexRow, styles.lh140Percent, styles.alignItemsEnd]}>
-                    <Text
-                        selectable={false}
-                        style={[styles.link, styles.ml2, styles.h4]}
-                    >
-                        {`${props.numberOfReplies} ${props.numberOfReplies === 1 ? props.translate('threads.reply') : props.translate('threads.replies')}`}
-                    </Text>
-                    <Text
-                        selectable={false}
-                        style={[styles.ml2, styles.textMicroSupporting]}
-                    >{`${props.translate('threads.lastReply')} ${props.datetimeToCalendarTime(props.mostRecentReply)}`}</Text>
+const ReportActionItemThread = (props) => {
+    const numberReplies = props.numberOfReplies > CONST.MAX_THREAD_REPLIES_PREVIEW ? `${CONST.MAX_THREAD_REPLIES_PREVIEW}+` : `${props.numberOfReplies}`;
```

Comment:
```suggestion
    const normalizedNumberOfReplies = props.numberOfReplies > CONST.MAX_THREAD_REPLIES_PREVIEW ? `${CONST.MAX_THREAD_REPLIES_PREVIEW}+` : `${props.numberOfReplies}`;
```

Reply:
NAB but I think this is more descriptive and easier to differentiate compared to just removing `of`